### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by topecongiro**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: Create a bug report for rustfmt
```
Comment (calebcartwright) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]: I wonder whether the average user would know which issue template to use :thinking: What if we made this a little more explicit like

```suggestion
about: Report rustfmt errors, failing to format inputs, etc.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior. If possible, please add a link to your project or a minimal working example.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Meta**
+
```
Comment (calebcartwright) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]: Perhaps an inline comment with instructions that will be visible to the user when creating the issue, but won't take up space in the issue once created

```suggestion
<!-- `rustfmt --version` -->

```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- rustfmt version:
```
Comment (calebcartwright) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]: Would it be worth asking whether users are running `rustfmt` vs. `cargo fmt`?

File: .github/ISSUE_TEMPLATE/formatting-issues.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+```
+
+**Expected output**
+
+```rust
+```
+
+**Meta**
+
+- rustfmt version: `rustfmt --version`
```
Comment (calebcartwright) [inline .github/ISSUE_TEMPLATE/formatting-issues.md:27]:
```suggestion
```suggestion
<!-- `rustfmt --version` -->

```
- rustfmt version:
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (topecongiro) [reply .github/ISSUE_TEMPLATE/bug_report.md:3]: Yeah, I agree. Thanks for the suggestion!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (topecongiro) [reply .github/ISSUE_TEMPLATE/bug_report.md:21]: That's good :+1:

**Pull request merged**